src/scss/components/_footer.scss

```diff
@@ -0,0 +1,107 @@
+.footer {
+    padding: 50px 0 65px;
```

Comment:
use relative sizes

Reply:
for width, height, padding, margin.

Reply:
But how would I actually use it if I say wanted to try and be pixel perfect with my layout.
Say if the design says the width is 100px;
Should I use a sass function to translate that value to rem? Or should I myself calculate the value (i.e. 100/16 as 6.25rem) and just put it in.